test/datasets/test_iwslt2016.py

```diff
@@ -78,7 +79,8 @@ def _get_mock_dataset(root_dir, split, src, tgt, valid_set, test_set):
     """
 
     base_dir = os.path.join(root_dir, DATASET_NAME)
-    outer_temp_dataset_dir = os.path.join(base_dir, f"2016-01/texts/{src}/{tgt}/")
+    temp_dataset_dir = os.path.join(base_dir, 'temp_dataset_dir')
```

Comment:
This addition of `temp_dataset_dir` is actually not necessary anymore now that you're deleting this directory at the end of this function. If the directory wasn't being deleted, this would be relevant since otherwise the caching logic would skip extracting the files (since the files exist within this dir). We can still keep this for consistency with the other tests.

Reply:
that's a good point. let me see where to remove the redundancy.

Reply:
OK, so I think let's keep it consistent with other datasets test. I just removed the deletion of temp_datset_dir.